src/System.Security.Cryptography.Algorithms/src/Internal/Cryptography/HashProviderDispenser.OSX.cs

```diff
@@ -235,6 +249,29 @@ public override unsafe byte[] FinalizeHashAndReset()
                 return hash;
```

Comment:
It looks like this got fixed in the mac version, but the Windows and OpenSsl/Unix versions still have parallel constructions.